.github/workflows/unittest.yml

```
@@ -0,0 +1,57 @@
+on:
+  pull_request:
+    branches:
+      - v1
+name: unittest
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.6', '3.7', '3.8']
```

We could drop 3.8 since it's not in the noxfile in the v1 branch

https://github.com/googleapis/python-logging/blob/7809d806dac776c385339dc71844d45245f0ce9a/noxfile.py#L100-L102

```suggestion
        python: ['3.6', '3.7']
```